blocks/forms/edit.js

```diff
@@ -103,7 +103,7 @@ const Edit = ({ attributes, setAttributes }) => {
 						src={iframeUrl}
 						style={{
 							border: '1px dashed #E0E0E0',
-							height: '500px',
+							height: formType === 'request' ? '1630px' : '600px',
```

Comment:
In my testing, the Booking form was about 320px tall so 600px adds lots of extra white space. I know it's hard to know for sure the height of each of those forms since I'm assuming it will depend on how each person has these configured but wondering if reducing it from 600 to 400 is a nicer middle ground?

Reply:
Yeah, it was around 590px for me on the front end – not sure why there’s a difference, but agreed that 400px seems like a nicer middle ground.